index.html

```diff
@@ -86,6 +86,192 @@ <h2 class="description">I’m a Software Developer</h2>
             </li>
         </ul>
     </section>
+    <ul class="works">
+        <li class="card card-primary">
+            <h2 class="card-header2">My Recent Works</h2>
+            <hr class="ruler" />
+            <img class="stretch" src="image_1.png" alt="A picture of a woman doing streching" />
+            <h3 class="card-header3">Multi-Post Stories</h3>
+            <p class="card-body1">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. has been the industry's standard dummy text ever
+                since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul>
+                <li class="lang1">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang1">
+                    <a href="#">html</a>
+                </li>
+                <li class="lang1">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang1">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project2">See Project</a>
+        </li>
+        <li class="card">
```

Comment:
- [ ] Same as above. you should replace `li` with `div` or `article` tag. and all the cases.